packages/module/src/load.cjs.ts

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2023 SolarWinds Worldwide, LLC.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+export function load(file: string): unknown {
+  /* eslint-disable-next-line @typescript-eslint/no-var-requires */
+  const required: unknown = require(file)
+
+  const fakeEsmDefaultExport =
```

Comment:
Hehe. Why is it "fake"?

Reply:
When TypeScript exports to CommonJS format it translates `export default` to a fake default like this, even though the format doesn't have that concept